components/dialog/dialog-mixin.js

```diff
@@ -162,6 +162,18 @@ export const DialogMixin = superclass => class extends RtlMixin(superclass) {
 		if (this.shadowRoot) this.shadowRoot.querySelector('.d2l-dialog-content').addEventListener('scroll', this._updateOverflow);
 	}
 
+	_checkForFocusableContentElem() {
```

Comment:
Doing this will remove the `tabindex="0"` from the dialog content and allow for calls to [__onAutoCloseFocus](https://github.com/BrightspaceUI/core/blob/main/components/dropdown/dropdown-content-mixin.js#L560) when a non-focusable item (e.g., text in a dropdown) is clicked to have the document as the `activeElement` and so return.

I'm wondering though if there's a better way to fix this in dropdown-mixin where we aren't relying on this function running. e.g., check for the dialog content being the active element (though that feels very specific).